README.md

```diff
@@ -28,7 +28,15 @@ pip install -r requirements.txt
 # Try it out
 python manim.py example_scenes.py OpeningManimExample
 ```
-
+### Mac OSX
+1. Install [FFmpeg],[LaTeX],[cairo](if you only want to run on CPU) in commandline using homebrew.
+  ```sh
+   brew install ffmpeg mactex cairo
+   ```
+2. Install manim using this command if you dont want to tinker with code.(virtualenv wrapper is recommended)
+  ```sh
+   pip3 install manimlib
```

Comment:
Please do not use pip to install `manimlib` now. The installed version is still the old version in the `cairo-backend` branch. The new version in the `master` branch has not been released to pypi. Currently, manim can only be used through cloning this repository.
Moreover, it is very inappropriate to introduce the installation method of the old version in the `cairo-backend` branch in this new version README.

Reply:
oh ,I didn't knew about that .I found in some tutorial that manimlib can be installed from pip but I didn't knew it was an older version.Thanks for pointing out.